pyshorteners/shorteners/__init__.py

```diff
@@ -72,7 +74,7 @@ def __init__(self, engine=Shorteners.SIMPLE, **kwargs):
             except AttributeError:
                 raise UnknownShortenerException(
                     'Please enter a valid shortener.'
-                    ' {} class does not exists'.format(self.engine)
+                    ' {} class does not exist'.format(self.engine)
```

Comment:
"class" is singular, so "exist" is grammatically correct. Do you still want me to revert it?